R/pkg/R/client.R

```diff
@@ -82,7 +82,7 @@ checkJavaVersion <- function() {
                    })
   javaVersionFilter <- Filter(
       function(x) {
-        grepl("java version", x)
+        grepl(" version", x)
```

Comment:
do we want the space " version" vs. "version" that is

Reply:
right, I debated about it. I think we should grep for the longest match as much as possible but I suppose the space is too common to be meaningful, so I don't feel strongly either way

Reply:
Is there any chance you have access to IBM jdk ? I'm just wondering if there is some other java version we are missing

Reply:
I managed to get it to work. I think our grep should be fine.
```
root@8c0bc3bb4123:~/ibm-java-x86_64-80# ./jre/bin/java -version
java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 8.0.5.15 - pxa6480sr5fp15-20180502_01(SR5 FP15))
IBM J9 VM (build 2.9, JRE 1.8.0 Linux amd64-64 Compressed References 20180425_385365 (JIT enabled, AOT enabled)
OpenJ9   - a7ffbfe
OMR      - a531219
IBM      - 59ef3dc)
JCL - 20180425_01 based on Oracle jdk8u171-b11
```

Reply:
great! interesting it's only broken on openjdk then